integration/gethnetwork/geth_network.go

```diff
@@ -255,11 +255,25 @@ func (network *GethNetwork) IssueCommand(nodeIdx int, command string) string {
 
 // StopNodes kills the Geth node processes.
 func (network *GethNetwork) StopNodes() {
+	var wg sync.WaitGroup
 	for _, process := range network.nodesProcs {
 		if process != nil {
-			_ = process.Kill()
+			tempProcess := process
+			wg.Add(1)
+			go func() {
```

Comment:
I think the idiomatic way of dealing with the closure here is to make process an argument of the anonymous function. I keep seeing it everywhere, i.e instead of creating `tempProcess`:

```
go func(process os.Process) {
...
}(process)

Or is there some reason that doesn't work here?

Reply:
Thanks, fixed.